modules/checkbox.js

```diff
@@ -0,0 +1,9 @@
+function checkboxCheck(todoItems) {
+  const checkboxes = document.querySelectorAll('.checkbox');
+  for (let i = 0; i < checkboxes.length; i += 1) {
+    if (checkboxes[i].checked) {
+      todoItems[i].completed = true;
```

Comment:
- [ ] Upon checking the checkbox, it fails to update to true in the local storage, I would encourage you to check if it is actually updating the local storage as this is a requirement.
![image](https://user-images.githubusercontent.com/86200356/223152195-9edf9441-6a28-41f7-84e8-caa12e657b23.png)